tests/test_scraper.py

```diff
@@ -0,0 +1,25 @@
+import os
+import os.path as op
+import pytest
+import pyvista
+from pyvista.utilities import Scraper
+
+
+def test_scraper(tmpdir):
+    pytest.importorskip('sphinx_gallery')
+    plotter = pyvista.Plotter(off_screen=True)
```

Comment:
Need to use `off_screen=True` @GuillaumeFavelier. `BackgroundPlotter` will not have this restriction but `Plotter` does.

I also added `sphinx_gallery` to `requirements_test`